android-todo-list/architecture-samples-main/app/src/main/java/com/example/android/architecture/blueprints/todoapp/util/StatsIgUtil.kt

```diff
@@ -43,4 +50,26 @@ object StatsIgUtil {
         return Statsig.checkGate(FEATURE_GATE_DELETE_TODO_ACCESS)
     }
 
+    /**
+     * The function to register the event without metadata in the Statsig portal.
+     *
+     * @param eventName Name of the event to be logged.
+     */
+    fun eventLogWithoutMetadata(eventName: String) {
+        Statsig.logEvent(eventName)
+    }
+
+    /**
+     * The function to register the event with metadata in the Statsig portal.
+     *
+     * @param eventName Name of the event to be logged.
+     */
+    fun eventLogWithMetadata(
```

Comment:
these two functions dont add anything - just use `Statsig.logEvent` directly in the callsite

Reply:
I will use Statsig.logEvent directly.

Reply:
Done in https://github.com/statsig-io/samples/pull/5